acceptance/tests/ticket_1123_facter_with_invalid_locale.rb

```diff
@@ -0,0 +1,7 @@
+test_name "ticket 1123  facter should not crash with invalid locale setting"
+confine :except, :platform => 'windows'
+agents.each do |host|
+  step "set an invalid value for locale and run facter"
+  on host,( "LANG=ABCD facter facterversion"), :acceptable_exit_codes => [ 0 , 2 ]
```

Comment:
Why is 2 an acceptable exit code? We really don't expect that to happen in this test, and not sure how to cause it.

Also, it'd be nice to confirm that something is printed to stdout, and nothing to stderr.